src/main/java/net/mcreator/workspace/elements/VariableType.java

```diff
@@ -19,16 +19,21 @@
 package net.mcreator.workspace.elements;
 
 import com.google.gson.annotations.SerializedName;
+import net.mcreator.blockly.BlocklyBlockUtil;
 import net.mcreator.generator.GeneratorConfiguration;
 import net.mcreator.generator.mapping.NameMapper;
 import net.mcreator.workspace.Workspace;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
+import java.awt.*;
 import java.util.Map;
 
 @SuppressWarnings("unused") public class VariableType {
+	private static final Logger LOG = LogManager.getLogger("Variable loader");

```

Comment:
You shouldn't name it `Variable loader` as methods are not called when they are loaded. A better name would be "Variable type", so the class' name.